internal/provider/provider.go

```diff
@@ -1215,6 +1215,7 @@ func Provider() *schema.Provider {
 
 			"aws_fsx_backup":                        fsx.ResourceBackup(),
 			"aws_fsx_lustre_file_system":            fsx.ResourceLustreFileSystem(),
+			"aws_fsx_data_repository_association":   fsx.ResourceDataRepositoryAssociation(),
```

Comment:
Should it be called ‘aws_fsx_data_repository_association’ or ‘aws_fsx_lustre_data_repository_association’?